datalab/settings.py

```diff
@@ -130,6 +130,10 @@ def get_list_from_env(variable, default=None):
 # AdminMiddleware is enabled.  The default value is 'default'.
 DRAMATIQ_TASKS_DATABASE = 'default'
 
+# AWS S3 Bitbucket
+DATALAB_OPERATION_OUTPUT_BUCKET = os.getenv('DATALAB_OPERATION_OUTPUT_BUCKET' , 'datalab-operation-output-bucket')
+AWS_ACCESS_KEY = os.getenv('AWS_ACCESS_KEY', '')
+AWS_PRIVATE_KEY = os.getenv('AWS_PRIVATE_KEY', '')
```

Comment:
Where are these settings being used? I'm used to boto3 expecting env variables set for `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`, as well as `AWS_DEFAULT_REGION`. It would be good to describe what is required to be set in the env in the readme.

Reply:
When I was testing locally boto3 was automatically pulling these from my ~/aws/.credentials locally so I think if we set them in the aws env it should work without explicitly using them?